### PR TITLE
Landing page wording + password modal

### PR DESCRIPTION
**Pull request by leventdem**

### CODE REVIEW

File: src/components/PasswordStrength/PasswordStrength.module.scss [hunk 1]
```diff
@@ -1,108 +1,123 @@
...
+        border-radius: 4px;
+        background-color: $color-warning;
+    }
+    .RectanglehighForce {
+        width: 260px;
+        height: 8px;
+        border-radius: 4px;
+        background-color: $color-success;
+    }
+}
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.module.scss]: You can configure vscode with `"files.insertFinalNewline": true,`

File: src/containers/Landing/Landing.jsx [hunk 2]
```diff
@@ -59,10 +59,10 @@ const Landing = () => (
 
       <div className={styles.Devices}>
         <div className={styles.text}>
-          <Typography type='title-landing'>Une synchronisation temps-réel entre appareils</Typography>
+          <Typography type='title-landing'>La synchronisation entre appareils en temps réel</Typography>
```
Comment (G-Ray) [inline src/containers/Landing/Landing.jsx]: il va falloir qu'on dise que c'est pour bientôt non ?

File: src/library/validators.js [hunk 3]
```diff
@@ -35,8 +34,12 @@ const checkPassword = str => {
   let force = null
   if (score === Object.keys(info).length) {
     force = 'high'
-  } else if (score >= 3 && (str.length >= 6)) {
-    force = 'medium'
+  } else if (score === 2 && (str.length >= 6)) {
+    force = 'medium-1'
```
Comment (G-Ray) [inline src/library/validators.js]: you can just return the value of force (an integer), and apply the corresponding css class like `force-1` `force-2` etc

File: src/library/validators.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ const isPassword = str => /^([a-zA-Z\d!?$#@()\-*])+$/.test(str)
 
 const isForceEnough = str => {
   const { force } = checkPassword(str)
-  return force !== 'low'
+  return force !== 0
```
Comment (G-Ray) [inline src/library/validators.js]: force > 0

File: src/library/validators.js [hunk 5]
```diff
@@ -33,11 +32,11 @@ const checkPassword = str => {
   const score = computeScore(info)
 
   let force = null
-  if (score === Object.keys(info).length) {
-    force = 'high'
-  } else if (score >= 3 && (str.length >= 6)) {
-    force = 'medium'
-  } else force = 'low'
+  if (str.length < 6 || score < 2) {
```
Comment (G-Ray) [inline src/library/validators.js]: easier: 
```js
force = str.length < 6 ? 0 : score
```

Except if you have a case in mind where str.length > 6 while score is < 2 ?

File: src/components/PasswordStrength/PasswordStrength.jsx [hunk 6]
```diff
@@ -88,23 +84,25 @@ const ForceBar = ({ force }) => {
...
-  force: 'low'
+  force: 0
 }
 const PasswordRules = ({ force, passwordInfo }) => {
-  const lockIcon = passwordInfo.force === 'low'
+  const lockIcon = passwordInfo.force === 0
     ? <Unlock
       size={30}
-      className={cx(styles.lockIcon, styles[force])}
+      className={cx(styles.lockIcon, styles[`force-${force}`])}
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.jsx:102]: ok good

File: src/library/validators.js (hunk 5 above)
Comment (leventdem) [reply src/library/validators.js]: Yes if there are 6 digit password for example, the score will be 1 but the password will not be accepted as we expect at least 6 characters and 2 rules.

File: src/library/validators.js (hunk 4 above)
Comment (leventdem) [reply src/library/validators.js]: ok, done

**Review by G-Ray — APPROVED** [review]

**Review by G-Ray — APPROVED** [review]

**Pull request merged**